src/js/angular/core/services/markdown/plugins/markdown-open-in-sparql-editor-plugin.js

```diff
@@ -0,0 +1,46 @@
+const DEFAULT_MARKDOWN_CONFIGURATION = {
+    iconClass: 'icon-sparql',
+    buttonStyle: "position: absolute; top: 0; right: 0;",
+    buttonClass: "btn btn-link btn-sm secondary"
+};
+
+/**
+ * This function enhances the rendering of fenced code blocks in markdown that contain SPARQL queries
+ * by appending an "open in SPARQL editor" button to each SPARQL code block.
+ *
+ * @param {Function} origRule The original rule function responsible for rendering code blocks.
+ * @param {Object} options The options to customize the rendering. These will be merged with the default configuration.
```

Comment:
It'd have been good if we had the options param model described.

Reply:
Done